python/cudf_polars/cudf_polars/testing/io.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+"""IO testing utilities."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import polars as pl
+
+
+def make_partitioned_source(
+    df: pl.DataFrame,
+    path: str | Path,
+    fmt: str,
```

Comment:
Could we use `Literal["csv", "ndjson", ...]`?

Reply:
Done.